stacks-signer/src/runloop.rs

```diff
@@ -176,12 +212,24 @@ impl<C: Coordinator> RunLoop<C> {
     /// Execute the given command and update state accordingly
     /// Returns true when it is successfully executed, else false
     fn execute_command(&mut self, command: &RunLoopCommand) -> bool {
+        let coordinator_info = calculate_coordinator(
```

Comment:
Right here we only need `coordinator_metadata`, so I think it would be better to have a function like `calculate_coordinator_metadata(...)`. And then at the top of `calculate_coordinator` we can call that function.